modules/upgrade-scanner.adoc

```diff
@@ -10,7 +10,6 @@ You can update Scanner to the latest version by using the `roxctl` CLI.
 .Prerequisites
 
 * If you deploy images from a private image registry, first push the new image into your private registry, and then replace your image registry for the commands in this section.
-* If you used Red Hat UBI-based images when you installed {product-title}, replace the image names for the commands in this section with the following UBI-based image names:
```

Comment:
This bullet item looked unfinished and so I deleted it. Please tell if I should put it back :-)